### DISCUSSION TITLE
Continued support for Ethers

### DISCUSSION THREAD

**Discussion by layinka** [wevm/wagmi#2489] [Ideas]
Hi,
Great work so far on wagmi. I also appreciate all the work that has gone on so far on Viem. 

I will like to ask if there will still be support for ethers js in future, i think it will be good if an option is proivded to use either viem or ethers js

**Comment by tmm** [comment]
Thanks! No plans to support ethers again at the moment. We think viem is better across most dimensions to build a library with and use (e.g. faster, type inference, smaller bundle, etc.). We can also iterate more quickly on wagmi and viem since the core team is the same.

Curious if there's anything about ethers you think is missing here?
